### PR TITLE
pi-agm 1.2.5, prompted by changes in the interval tactic

### PR DESCRIPTION
**Pull request by ybertot**

### CODE REVIEW

File: released/packages/coq-pi-agm/coq-pi-agm.1.2.5/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+the calculus-based proofs for an abstract view of the algorithm, where all
+numbers are real numbers.  A second file describes how to approximate all
+computations using large integers.  Other files describe the second algorithm
+which is close to the one used in mpfr, for instance.
+
+The whole development can be used to produce mathematically proved and
+formally verified approximations of PI."""
+url {
+  src: "https://github.com/ybertot/pi-agm/archive/v1.2.5.zip"
+  checksum: "sha256=22db29bf274a58cae18966ca2e0126811eb7223ef58fb23486727e1cd16162a1"
```
Comment (clarus) [inline released/packages/coq-pi-agm/coq-pi-agm.1.2.5/opam:35]:
```suggestion
  checksum: "sha256=0cb93b44a7198bd157cb16a661b70e0e2ccf858bdbf142815740136b69b52627"
```

**Comment by ybertot** [comment]
I did download the file, but my download failed and I put the checksum of the failure report.  Thanks for checking.

File: released/packages/coq-pi-agm/coq-pi-agm.1.2.5/opam [hunk 2]
```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+name: "coq-pi-agm"
+version: "1.2.5"
```
Comment (palmskog) [inline released/packages/coq-pi-agm/coq-pi-agm.1.2.5/opam:3]:
```suggestion
opam-version: "2.0"
```
Name and version are already given by the file name.

File: released/packages/coq-pi-agm/coq-pi-agm.1.2.5/opam [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+maintainer: "yves.bertot@inria.fr"
+
+homepage: "http://www-sop.inria.fr/members/Yves.Bertot/"
+bug-reports: "yves.bertot@inria.fr"
+license: "CeCILL-B"
```
Comment (palmskog) [inline released/packages/coq-pi-agm/coq-pi-agm.1.2.5/opam:8]:
```suggestion
license: "CECILL-B"
```
To match SPDX identifier exactly.

File: released/packages/coq-pi-agm/coq-pi-agm.1.2.5/opam [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+build: [["coq_makefile" "-f" "_CoqProject" "-o" "Makefile" ]
+       [ make "-j" "%{jobs}%" ]]
+install: [ make "install" "DEST='%{lib}%/coq/user-contrib/pi_agm'" ]
```
Comment (palmskog) [inline released/packages/coq-pi-agm/coq-pi-agm.1.2.5/opam:11]: Is there an obvious reason for the logpath (`agm`) to diverge from the installed directory name (`pi_agm`)? To my knowledge, this means that all modules will have to be accessed by consumers of the library using `pi_agm`.

File: released/packages/coq-pi-agm/coq-pi-agm.1.2.5/opam (hunk 2 above)
Comment (palmskog) [inline released/packages/coq-pi-agm/coq-pi-agm.1.2.5/opam:3]:
```suggestion
```

**Pull request merged**